src/app/rpmostree-builtin-rebase.c

```diff
@@ -33,9 +33,13 @@
 static char *opt_osname;
 static gboolean opt_reboot;
 static gboolean opt_skip_purge;
+static char * opt_branch;
+static char * opt_remote;
 
 static GOptionEntry option_entries[] = {
   { "os", 0, 0, G_OPTION_ARG_STRING, &opt_osname, "Operate on provided OSNAME", "OSNAME" },
+  { "branch", 'b', 0, G_OPTION_ARG_STRING, &opt_branch, "Rebase to branch BRANCH from the current remote", "BRANCH" },
```

Comment:
Maybe just `Rebase to branch BRANCH`? Otherwise it sounds a bit like `b` and `-m` are mutually exclusive options. Hmm, or maybe `Rebase to branch BRANCH (use -m to also change remote)` ?